src/main/scala/nl.knaw.dans.easy.archivebag/EasyArchiveBag.scala

```diff
@@ -148,7 +148,7 @@ object EasyArchiveBag extends Bagit5FacadeComponent with DebugEnhancedLogging {
       response <- Using.fileInputStream(file)
         .map(fileStream => {
           s.http(s.storageDepositService.toURI.resolve("bags/").resolve(s.bagId.toString).toASCIIString)
-            .copy(connectFunc = InputStreamBodyConnectFunc(fileStream))
+            .copy(connectFunc = InputStreamBodyConnectFunc(fileStream, Option(file.length())))
```

Comment:
Shouldn't `file.length()` be called without parentheses, as it does not change any state?